models/settings_helpers.go

```diff
@@ -295,6 +332,13 @@ func ValidateSettings(params *ChangeSettingsParams) error {
 		return fmt.Errorf("Both pmm_public_address and remove_pmm_public_address are present.") //nolint:golint,stylecheck
 	}
 
+	if (EmailAlertingSettings{}) != params.EmailAlertingSettings && params.RemoveEmailAlertingSettings {
```

Comment:
Looks like it is written backwards
```suggestion
	if params.EmailAlertingSettings != (EmailAlertingSettings{}) && params.RemoveEmailAlertingSettings {
```

Or maybe we can make `params.EmailAlertingSettings` pointer and check for `nil`?

Reply:
reversed thanks

Reply:
But can we use pointers and check for `!= nil` ?

Reply:
sorry didn't have time to do that yesterday, but have changed them to pointers now